src/clarity-angular/wizard/wizard-stepnav-item.ts

```diff
@@ -49,7 +49,7 @@ export class WizardStepnavItem {
 
     public get isDisabled(): boolean {
         this.pageGuard();
-        return this.page.disabled;
+        return this.page.disabled || this.navService.wizardStopNavigation || this.navService.wizardDisableStepnav;
```

Comment:
These will always have a value right?
IIR correctly this evaluates each expression and returns the first true value or the last false value. But, if they are all null it will return that.